nav2_rviz_plugins/include/nav2_rviz_plugins/navigation_dialog.hpp

```diff
@@ -49,6 +49,7 @@ private slots:
   // time out for wait_for_action_server and send goal (seconds)
   int timeout = 3;
 
+  int timeout = 3;
```

Comment:
Redundant . Remove duplicate

Reply:
I wanted to try to use the Github for doing commits and fixing conflicts and this happened. Lesson learned. I will stick with git :)